org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtBuildReporter.scala

```diff
@@ -87,7 +87,7 @@ private[zinc] class SbtBuildReporter(project: IScalaProject) extends xsbti.Repor
       val markerPos = MarkerFactory.RegionPosition(offset, identifierLength(pos.lineContent, pos.pointer), line)
       BuildProblemMarker.create(resource, severity, msg, markerPos)
     } else
-      logger.info("suppressed error in Java file: %s".format(msg))
+      logger.info(s"suppressed error in Java file ${resource.getFullPath}:$line: $msg")
```

Comment:
I agree we need to do something better here, but I don't have a good solution right now. Here's why we don't report it: Java sources are compiled by the Eclipse Java compiler. It will report its own error markers, later -- we'd be duplicating those.

Here's why we should report it: the Scala compiler reads those Java files because it needs to compile Scala sources. If there's an error in a Java file it *won't produce bytecode*. That's not a problem, as usually the Eclipse Java compiler already reported those errors. Now the beauty of this is that the Scala parser for Java files is a bit different from the Java one. You might have a correct Java file that leads to no errors (Eclipse Java thinks it's ok, the Scala compiler does not -- and won't compile Scala sources either) *and no bytecode at all*.

It's a catch-22.